packages/frontend/src/blockchain/hooks/useAuctionState.ts

```diff
@@ -64,12 +65,17 @@ export const useContractState = () => {
   return { state: data as ContractState, isLoading }
 }
 
-function getStateUsingWallet(account: Hex | undefined, chainId: number | undefined, state: AuctionState) {
+function getStateUsingWallet(
+  account: Hex | undefined,
+  userChainId: number | undefined,
+  chainId: number,
+  state: AuctionState,
+) {
```

Comment:
I think more than 3 arguments may call for a dict of params, especially that two of them are `chainId`